demosplan/DemosPlanCoreBundle/Logic/Permission/AccessControlService.php

```diff
@@ -209,15 +210,42 @@ public function enablePermissionCustomerOrgaRole(string $permissionToEnable, Cus
                 continue;
             }
 
+            $updatedOrga = $this->addPermissionBasedOnOrgaType($permissionToEnable, $role, $orgaInCustomer, $customer, $dryRun);
+
+            if (null !== $updatedOrga) {
+                $updatedOrgas[] = $updatedOrga;
+            }
+        }
+
+        return $updatedOrgas;
+    }
+
+    /**
+     * $orgatype = getorgatype.
```

Comment:
Wht do you mean by this dockblock? I do not understand

Reply:
@mussbach done ce8d4b56ccb5862b11ef0ea3875624a2c7b4d3bc